modules/deployment-scripts/create-kv-certificate/create-kv-cert.sh

```diff
@@ -1,54 +1,64 @@
 #!/bin/bash
 set -e
+initialDelay="${initialDelay:-5}"
+retryMax="${retryMax:-5}"
+certName="${certName:-default-cert}"
+certCommonName="${certCommonName:-default}"
+validity="${validity:-12}"
+akvName="${akvName:-keyvault}"
+issuerName="${issuerName:-}"
+reuseKey="${reuseKey:-true}"
+retrySleep="${retrySleep:-5}
```

Comment:
⚠️ **[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Did you forget to close this double quoted string? [SC1078](https://github.com/koalaman/shellcheck/wiki/SC1078)